thavalon-webapp/src/components/Register.tsx

```diff
@@ -1,32 +1,88 @@
-import React, {useEffect, useState} from 'react';
+import React, {useState} from 'react';
 import ReactModal from 'react-modal';
 import { useForm } from 'react-hook-form';
 import "./modal.scss";
-import { register_user } from '../utils/account_utils';
+import AccountManager, { HttpResponse } from '../utils/accountManager';
+import { Redirect } from 'react-router-dom';
+
+type RegisterProps = {

```

Comment:
Let's use interface here instead of type. Interface is used for defining the shape of an object, while type is mostly used for unioning or intersections with other interfaces.